Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -1570,7 +1570,9 @@ def revoke_security_group_ingress_command(args, aws_client):
         kwargs.update({'SourceSecurityGroupName': args.get('sourceSecurityGroupName')})
 
     response = client.revoke_security_group_ingress(**kwargs)
-    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200 and response['Return']:
```

Comment:
is the `response['Return']` condition relevant for all commands?

Reply:
relevant to some of the commands. fixed where needed: https://docs.aws.amazon.com/cli/latest/reference/ec2/index.html#cli-aws-ec2